CDCgov/prime-reportstream
# Prototype for testing transport layer using Docker containers

This PR brings in the [test containers dependency](https://java.testcontainers.org/) to spin up a live local server in a Docker container acting as a receiver to test our REST transport layer.

Test Steps:
1. Run the integration tests. Take a look at the logs to see the new test and how it works.

## Changes
- Added test containers dependency
- Added test
- Added Dockerfile to spin up fake receiver web server
- Added simple Python web server file to use in Dockerfile

## Checklist

### Testing
- [ ] ./gradlew testIntegration

### Process

## Linked Issues
- Fixes #issue
